packages/client/shared/rruleUtil.ts

```diff
@@ -1,21 +1,33 @@
-import {datetime} from 'rrule'
+import dayjs, {Dayjs} from 'dayjs'
+import customParsePlugin from 'dayjs/plugin/customParseFormat'
+import timezonePlugin from 'dayjs/plugin/timezone'
+import utcPlugin from 'dayjs/plugin/utc'
+import {RRule} from 'rrule'
 
-export const getRRuleDateFromJSDate = (date: Date) => {
-  return datetime(
-    date.getFullYear(),
-    date.getMonth() + 1,
-    date.getDate(),
-    date.getHours(),
-    date.getMinutes()
-  )
+dayjs.extend(customParsePlugin)
+dayjs.extend(utcPlugin)
+dayjs.extend(timezonePlugin)
+
+// the RRule package requires dstart to be a date object set to a negative UTC offset. It's ugly!
```

Comment:
+1 I don't see where we set a negative UTC offset here or what that means.

Reply:
Same as the old helper functions in this file, the rrule package stores `dtstart` as a Date, but interprets the local time as UTC time. e.g. it'll create a Date of 1/1/2020 8:00AM PST and then interpret that as 1/1/2020 8:00AM UTC. It's pants-on-head stupid. https://github.com/jkbrzt/rrule/issues/501#issuecomment-1146513557.